commands/status.go

```diff
@@ -46,13 +51,14 @@ func (res *statusResult) String() string {
 }
 
 type statusCommand struct {
+	getNetworkInfo bool
```

Comment:
We don't set it anywhere.

Reply:
yeah, my bad. I had a flag before, but then realized that we should show this allways.